ramanujantools/matrix.py

```diff
@@ -33,6 +33,21 @@ def __call__(self, substitutions: Dict) -> Matrix:
         Substitutes symbols in the matrix, in a more math-like syntax.
         """
         return self.subs(substitutions)
+    
+    def as_latex(self, print_bool: bool = True) -> None:
+        """
+        Prints the matrix as a latex string.
+        Args:
+            print_bool: if `True`, prints the latex string to the console (set to False for testing).
```

Comment:
Remove the `print_bool`.
Printing should never occur from inside the functions, always let the user decide what he prints.
If the function returns the string, the user will be able to print it himself easily.

Reply:
Good point